building/build-debs/homeworld-admin-tools/build-package.sh

```diff
@@ -1,6 +1,28 @@
 #!/bin/bash
-set -eu
+set -e -u
+rm -f resources/GIT_VERSION
+
+git update-index --refresh -q
+set +e
+git diff-index --quiet HEAD --
+GIT_DIFF_RETVAL=$?
+set -e
+
+GIT_ROOT="$(git rev-parse --show-toplevel)"
+set +e
+u="$(git ls-files --exclude-standard --others "${GIT_ROOT}")" && test -z "$u"
+GIT_LS_RETVAL=$?
+set -e
+
+git rev-parse HEAD | tr -d '\n' >resources/GIT_VERSION
+if [[ $GIT_DIFF_RETVAL -ne 0 ]] || [[ $GIT_LS_RETVAL -ne 0 ]]
+then
+    echo -n "-dirty" >>resources/GIT_VERSION
+fi
+echo >>resources/GIT_VERSION
+
 source ../common/package-build-helpers.sh
 
 upstream "debian-9.2.0-amd64-mini.iso"
 build
+rm resources/GIT_VERSION
```

Comment:
Might be worth doing this in a trap handler, so that it happens even if the build fails.